src/Abc.Zebus/SubscriptionHandling/SubscriptionHandler.cs

```diff
@@ -0,0 +1,27 @@
+using Abc.Zebus.Directory;
+
+namespace Abc.Zebus.SubscriptionHandling
```

Comment:
Maybe `Abc.Zebus.Subscriptions` would be a better namespace name? Or even use `Abc.Zebus.Directory`?

Reply:
Agreed that `Subscriptions` is a better name for the namespace